pkg/client/kubernetes/applier.go

```diff
@@ -163,7 +156,7 @@ var DefaultMergeFuncs = MergeFuncs{
 				break
 			}
 
-			ports := []interface{}{}
+			var ports []interface{}
```

Comment:
minor nit suggestion
```suggestion
			ports := make([]interface{}, len(newPorts))
```